frontend/views/containers/settings/Appearence.vue

```diff
@@ -1,15 +1,15 @@
 <template lang='pug'>
   .settings-container
-    h1.settings-title.is-size-4
+    h1.settings-title
       i18n Appearence
 
-    section.section
-      h2.settings-subtitle.is-size-5 Theme
+    section.card
+      h2.settings-subtitle Theme
 
       SelectorTheme
 
-    section.section
-      h2.settings-subtitle.is-size-5 Text settings
+    section.card
+      h2.settings-subtitle Text settings
```

Comment:
Found this strange bug in the size of the cards when going to settings appearance:

![bug_app](https://user-images.githubusercontent.com/14869087/64179812-aa899580-ce5b-11e9-888a-40fe24b737b8.gif)